website/docs/core-concepts/components/remote-state.md

```diff
@@ -158,7 +159,7 @@ module "vpc_flow_logs_bucket" {
 In the `components/terraform/infra/vpc/vpc-flow-logs.tf` file, configure the `aws_flow_log` resource for the `vpc` Terraform component to use the
 remote state output `vpc_flow_logs_bucket_arn` from the `vpc-flow-logs-bucket-1` Atmos component:
 
-```hcl title="components/terraform/infra/vpc/remote-state.tf"
```

Comment:
this is correct: this comes from : https://github.com/cloudposse/terraform-aws-components/blob/master/modules/vpc/remote-state.tf

Reply:
Yes, but [in the doc on the Atmos website](https://atmos.tools/core-concepts/components/remote-state), this code block's header reads `components/terraform/infra/vpc/remote-state.tf` when the paragraph before it says:

> In the `components/terraform/infra/vpc/vpc-flow-logs.tf file`, configure the `aws_flow_log` resource

Is the `resource "aws_flow_log"` block in this example supposed to be defined in `remote-state.tf`?